### PR TITLE
Fix Docs GitHub Action with ignore list and update

### PR DESCRIPTION
**Pull request by joelhans**
##### Summary

I've noticed that the Docs GitHub Action always fails due to failing to check the link `https://pi-hole.net`. There's two problems with this. First, it shouldn't always be checking links in the Pi-hole guide—that's a bug. Second, the link is functional, so I'm choosing to ignore the failure.

##### Component Name

area/ci

### CODE REVIEW

File: .mlc_config.json [hunk 1]
```diff
@@ -1,4 +1,9 @@
 {
+    "ignorePatterns": [
+		{
+			"pattern": "^https://pi-hole.net"
```
Comment (prologic) [inline .mlc_config.json:4]: Technically if this is a regex is should probably be `^https\:\/\/pi-hole\.net`

**Review by prologic — DISMISSED** [review]

File: .mlc_config.json (hunk 1 above)
Comment (joelhans) [reply .mlc_config.json]: Cool, thanks! I saw some errors in VSCode related to having the escape in front of the `:` and `.`. Do you know if this matters? It stopped being mad at me if the regex is `^https:\/\/pi-hole.net`.

**Review by prologic — APPROVED** [review]

**Comment by joelhans** [comment]
@Ferroin Mind looking at this? It would be nice to fix the **Docs** GHA so that we can more robustly ensure that we're not merging broken links into the docs.

**Review by Ferroin — APPROVED** [review]

**Comment by Ferroin** [comment]
@joelhans Sorry about the delay in getting this approved, I seem to have missed it somehow...

**Comment by joelhans** [comment]
@Ferroin No worries! I forgot about it myself until I was tracking down some broken links yesterday. Looks like James' approval isn't valid any more, so we need one more. Have anyone in mind?

**Comment by Ferroin** [comment]
At this point there really isn’t anyone else other than me, so I’ll just merge it.

**Pull request merged**